tasks/configure.yml

```diff
@@ -41,16 +41,16 @@
   file:
     name: "{{ pdns_rec_config['include-dir'] }}"
     state: directory
-    owner: "root"
-    group: "root"
-    mode: 0755
+    owner: "{{ pdns_fileowner.user }}"
+    group: "{{ pdns_fileowner.group }}"
+    mode: 0750
   when: "pdns_rec_config['include-dir'] is defined"
 
 - name: Generate the PowerDNS Recursor Lua config-file
   copy:
     dest: "{{ pdns_rec_config_lua }}"
     content: "{{ pdns_rec_config_lua_file_content }}"
-    owner: "root"
+    owner: "{{ pdns_rec_user }}"
```

Comment:
```suggestion
    owner: "{{ pdns_fileowner.user }}"
```

Reply:
I have changed the hash to pdns_rec_file_owner. 
There are now two different entries, one for the owner of the configuration file, and one for the user that runs the service. 
Question from me: should these be the same in all cases, or would there ever be a case when they might be different? If they are always the same, we can merge the two variables.

Reply:
They don't have to be the same. Users can have reasons to e.g. have it owned by `root:adm`

Reply:
Updated with the correct entity.